docker/container.go

```diff
@@ -212,7 +212,13 @@ func (c *Container) createContainer(imageName string) (*dockerclient.Container,
 
 	_, err = c.client.CreateContainer(config, c.name)
 	if err != nil && err.Error() == "Not found" {
-		err = c.pull(config.Image)
+		logrus.Debugf("Not Found, pulling image %s", config.Image)
```

Comment:
I've fixed real quick the integration test that was failing :stuck_out_tongue_closed_eyes: